rlclientlib/utility/http_helper.cc

```diff
@@ -1,15 +1,19 @@
 #include "http_helper.h"
+#include "constants.h"
 
 #include <chrono>
 
 namespace reinforcement_learning {
   namespace utility {
-    web::http::client::http_client_config get_http_config() {
+    web::http::client::http_client_config get_http_config(const utility::configuration &cfg) {
       web::http::client::http_client_config config;
 
-      config.set_validate_certificates(false);
-      // Set a timeout for network requests to avoid hanging too long because of network issues.
-      config.set_timeout(std::chrono::seconds(2));
+      config.set_validate_certificates(cfg.get_bool(name::HTTP_CLIENT_DISABLE_CERT_VALIDATION, false));
+      int timeout = cfg.get_int(name::HTTP_CLIENT_TIMEOUT, 30);
```

Comment:
```suggestion
      const auto timeout = cfg.get_int(name::HTTP_CLIENT_TIMEOUT, 30);
```

Reply:
can't use const here as I change it down